SpiderKeeper/app/spider/controller.py

```diff
@@ -676,3 +388,13 @@ def service_stats(project_id):
     project = Project.find_project_by_id(project_id)
     run_stats = JobExecution.list_run_stats_by_hours(project_id)
     return render_template("server_stats.html", run_stats=run_stats)
+
+
+@app.route("/favicon.ico")
```

Comment:
Currently this method simply return empty string, this does not look like proper handing